docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: '3.8'
+name: ecom-backend
+services:
+  api:
+    build: .
+    ports:
+      - "8080:8080"
+    env_file:
+      - .env.production
+    volumes:
+      - ./src/logs:/usr/src/app/src/logs
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)